recordedit/form.controller.js

```diff
@@ -433,6 +445,7 @@
         }
 
         function createRecord(column) {
+            // TODO: wrapper function with catch here
```

Comment:
Remove the check for appURL. Have appLink function in common check for `undefined`